projects/ama-testing/src/e2e/pages/process-properties.card.ts

```diff
@@ -79,6 +79,12 @@ export class ProcessPropertiesCard extends GenericPage {
         await super.click(connectorActionOption);
     }
 
+    async checkConnectorInList(connectorName: string) {
```

Comment:
Sorry, but I don't think it belongs here. In the activiti (O.S) distribution, we don't have a connector selector. We have it only in the alfresco version. This one rather looks like that we need to extend the `ProcessPropertiesCard` page object in the alfresco repo with this method `AdvancedProcessPropertiesCard`. @rdiacenco, what do you think?

We should put methods into the activiti repo if that method is used in this repository at least. If the new method was to be used only in alfresco's e2e, then that means the method does not belong to the activiti repository.

Reply:
Yes, this is right, I am the one creating the confusion, since I wrongly added, from the beginning, in Activiti, methods related to Enterprise version.
I would rename this method to 'isConnectorInList', since it returns a boolean and it doesn't perform a check.